src/js/letters/components/Address.jsx

```diff
@@ -116,31 +104,32 @@ class Address extends React.Component {
     // This will be changed once we pull the real country list from the
     // address endpoint.
     let selectedCountry;
-    if (this.props.value.country === undefined && addressType === 'MILITARY') {
+    if (this.props.address.country === undefined && addressType === 'MILITARY') {
       selectedCountry = 'USA';
-    } else if (this.props.value.country === 'US') {
+    } else if (this.props.address.country === 'US') {
       selectedCountry = 'USA';
     } else {
-      selectedCountry = this.props.value.country;
+      selectedCountry = this.props.address.country;
     }
 
+
     const adjustedStateNames = this.adjustStateNames(STATE_CODE_TO_NAME, militaryStateNames, cityField);
 
     const stateProvince = selectedCountry === 'USA'
-      ? (<ErrorableSelect errorMessage={this.isValidAddressField(this.props.value[stateField]) ? undefined : 'Please enter a valid state/province'}
+      ? (<ErrorableSelect errorMessage={this.isValidAddressField(this.props.address[stateField]) ? undefined : 'Please enter a valid state/province'}
```

Comment:
The `stateField` and `cityField` stuff has changed in @cvalarida latest PR he merged, so this will change once you rebase/pull from master next.